src/pages/_app.tsx

```diff
@@ -15,7 +15,19 @@ const MyApp = ({ Component, pageProps }: AppProps) => {
     Router.events.on("routeChangeStart", () => setIsLoading(true));
     Router.events.on("routeChangeComplete", () => setIsLoading(false));
     Router.events.on("routeChangeError", () => setIsLoading(false));
-  }, [Router.events]);
+
+    const unobtrusiveDataReload = () => {
+      Router.replace(window.location, undefined, { scroll: false }).catch(
+        () => {
+          throw new Error("Could not do unobtrusiveDataReload");
```

Comment:
Users are unlikely to ever hit that error, but if they do currently there’s nothing to handle it.

Reply:
Yeah eslint was yelling at me so I just felt compelled to do something. I forgot there's nothing to catch it. Should I just ignore it or should I do something more useful with it?

Reply:
Nah I think this is fine.

I do think we should have some top-level error handler that displays a generic error page, but that’s orthogonal to what you’re doing here.